src/Entity/Track/Version.php

```diff
@@ -142,4 +151,43 @@ public function setNegativeElevation(int $negativeElevation): void
     {
         $this->negativeElevation = $negativeElevation;
     }
+
+    public function getRatings()
+    {
+        return $this->ratings;
+    }
+
+    /**
+     * Get overall rating by all users
+     *
+     * @return float
+     */
+    public function getOverallRating(): float
+    {
+        if ($this->ratings->isEmpty()) {
+            return 0;
+        }
+
+        $sum = 0;
+        $total = $this->ratings->count();
+
+        /**
+         * @var $rating Rating
+         */
+        foreach ($this->ratings as $rating) {
+            $sum += $rating->getRating();
+        }
+
+        return round($sum / $total, 1);
```

Comment:
round() should be done in the template.
We may need exact number to order track by rating